story/src/2-Table.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Table } from '../../frontend/src/component/ui/Table';
+import { array, number, select } from '@storybook/addon-knobs';
+
+export default {
+  title: 'UI/Table',
+  component: Table
+};
+
+function alignSelector(label: string) {
+  return select(
+    label,
+    {
+      center: 'center',
+      end: 'end',
+      justify: 'justify',
+      left: 'left',
+      'match-parent': 'match-parent',
+      right: 'right',
+      start: 'start'
+    },
+    'left'
+  );
+}
+
+export const table = () => {
+  const headers = array('Header Columns', ['Long Link', 'Alias']);
+  return (
+    <Table
+      headers={headers}
+      rows={new Array(number('Number of rows', 2))
+        .fill(0)
+        .map((_, i) => headers.map((_, j) => `data ${i}-${j}`))}
```

Comment:
I am confused by what's going on here. Can you explain what's going on or extract the logic using helper functions?

Reply:
I am simply allowing the table to have the number of rows user specifies within the storybook. I can make it into a helper function if that makes it easier to understand.